src/db/seeders/user.js

```diff
@@ -0,0 +1,24 @@
+const successfullSignup = {
```

Comment:
Typo - `successfullSignup`.

Did you mean `successfulSignup`?

Reply:
Thanks for pointing this out. I'll correct it.